spyder/utils/icon_manager.py

```diff
@@ -162,8 +162,8 @@
     'error':                   [('fa.times-circle',), {'color': 'darkred'}],
     'warning':                 [('fa.warning',), {'color': 'orange'}],
     'todo':                    [('fa.exclamation',), {'color': '#3775a9'}],
-    'ipython_console':         [('spyder.ipython-logo-alt',), {'color': MAIN_FG_COLOR}],
-    'ipython_console_t':       [('spyder.ipython-logo-alt',), {'color':'gray'}],
+    'ipython_console':         [('mdi.console',), {'color': MAIN_FG_COLOR}],
+    'ipython_console_t':       [('mdi.console',), {'color':'gray'}],
```

Comment:
Fix PEP 8 warning

```suggestion
    'ipython_console_t':       [('mdi.console',), {'color': 'gray'}],
```

Reply:
To address this, please press the button `Commit suggestion` above and after that `Commit changes`.